### PR TITLE
[INLONG-8087][Manager]  Add definition of Inlong tenant table

### LINKED ISSUE #8087: [Feature][Manager] Add definition of Inlong tenant table
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by vernedeng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-XYZ][Component] Title of the pull request

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/inlong/issues) number)*

- Fixes #8087 
- Parent #7914 

### Motivation

*Explain here the context, and why you're making that change. What is the problem you're trying to solve?*

### Modifications

*Describe the modifications you've done.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
please update the `changes-1.8.0.sql` at the same time.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongTenantEntityMapper.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.dao.mapper;
+
+import org.apache.inlong.manager.dao.entity.InlongTenantEntity;
+
+public interface InlongTenantEntityMapper {
```
Comment (healchow) [inline inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongTenantEntityMapper.java:22]: Please add `@Repository` annotation for the Mapper interface.

File: inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql [hunk 2]
```diff
@@ -791,5 +791,29 @@ VALUES ('audit_sdk_collect', 'SDK', 0, '1'),
        ('audit_sort_kudu_output', 'KUDU', 1, '26');
 
 -- ----------------------------
+-- Table structure for inlong_tenant
+-- ----------------------------
+CREATE TABLE IF NOT EXISTS `inlong_tenant`
```
Comment (healchow) [inline inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql:796]: Please add this table into the `manager-test` module.

Comment (vernedeng) [reply inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql:796]: I have added this table into the sql in h2 dictionary.

File: inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongTenantEntityMapper.java (hunk 1 above)
Comment (vernedeng) [reply inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/mapper/InlongTenantEntityMapper.java:24]: fixed, thx

**Review by fuweng11 — APPROVED** [review]

**Review by healchow — APPROVED** [review]

File: inlong-manager/manager-web/sql/changes-1.8.0.sql [hunk 3]
```diff
@@ -25,3 +25,22 @@ USE `apache_inlong_manager`;
 
 ALTER TABLE inlong_group
     CHANGE lightweight inlong_group_mode tinyint(1) DEFAULT 0 NULL COMMENT 'Inlong group mode, Standard mode: 0, DataSync mode: 1';
+
```
Comment (dockerzhang) [inline inlong-manager/manager-web/sql/changes-1.8.0.sql:28]: It's better to add more annotations to show the purpose of this change. Like `For support multi-tenant management in InLong, see https://github.com/apache/inlong/pull/8090`.

Comment (vernedeng) [reply inlong-manager/manager-web/sql/changes-1.8.0.sql:28]: fixed, thx.

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**